src/components/Header.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+Header.propTypes = {
+  date: PropTypes.object.isRequired,
+}
+
+export default function Header({ date }) {
+  const month = [
+    'January',
+    'February',
+    'March',
+    'April',
+    'May',
+    'June',
+    'July',
+    'August',
+    'September',
+    'October',
+    'November',
+    'December',
+  ]
+
+  const year = date.getFullYear()
+  const monthNumber = date.getMonth()
+  const newMonth = month[monthNumber]
```

Comment:
good logical thinking to extract the month from the date object.
for less code (without the month array) and modern ECMAScript code look at toLcaleString method.
`const month = date.toLocaleString('default', { month: 'long' });` => 'September'

[short explanation to this topic](https://stackoverflow.com/a/18648314/6086226)